adagrams/game.py

```diff
@@ -1,11 +1,104 @@
+from string import ascii_uppercase
+from random import randint , choice
+
+
+
 def draw_letters():
-    pass
+    letter_pool = {
```

Comment:
`letter_pool` is constant so it should be named with capital letters like `LETTER_POOL`

Reply:
Ooh you're right, I missed that you're subtracting the values. Not a constant!